presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -84,6 +88,23 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                                 .map(name -> ((String) name).toLowerCase(ENGLISH))
                                 .collect(Collectors.toList())),
                         value -> value),
+                new PropertyMetadata<>(
+                        ORC_BLOOM_FILTER_COLUMNS,
+                        "orc bloom index columns",
+                        typeManager.getType(parseTypeSignature("array(varchar)")),
+                        List.class,
+                        ImmutableList.of(),
+                        false,
+                        value -> ((Collection<?>) value).stream()
```

Comment:
This is repeated above, maybe add a helper. (it's also confusing since it does the same thing but is slightly different code)